great-idea/index.html

```diff
@@ -16,7 +16,66 @@
 
   <body>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
+    <div class="container">
+      
+           <div class="navigation">
+               <nav>
+                  <a href="#">Services</a>
+                  <a href="#">Product</a>
+                  <a href="#">Vision</a>
+                  <a href="#">Features</a>
+                  <a href="#">About</a>
+                  <a href="#">Contact</a>
+               </nav>
+  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
```

Comment:
Need to work on indentation best practices, here the image tag is out of place and properly indented with the rest of the code. and is like this in other places of your code.